project1/application.py

```diff
@@ -20,7 +22,21 @@
 engine = create_engine(os.getenv("DATABASE_URL"))
 db = scoped_session(sessionmaker(bind=engine))
 
+names = {}
+
 
 @app.route("/")
 def index():
-    return "Project 1: TODO"
+    return render_template("index.html")
+
+
+@app.route("/register", methods=["POST", "GET"])
+def register():
+    if request.method == "POST":
+        uname = request.form.get("uname")
+        pwd = request.form.get("pwd")
+        email = request.form.get("email")
+        print(uname+"    "+pwd + " "+email)
+        names[email] = [uname, pwd]
```

Comment:
Avoid storing data on the application itself. All the user data should be in the database.